### PR TITLE
Replaces workload table with text data

### PR DESCRIPTION
**Pull request by bhanvimenghani**
This pr solves issue #48

### CODE REVIEW

**Comment by bharathappali** [comment]
@bhanvimenghani Thanks for raising the PR. Is there a working kruize image where I can see these changes? or should I be running it locally to check?

**Comment by bhanvimenghani** [comment]
@bharathappali There is no image created for this, if u want I can create now

**Comment by bhanvimenghani** [comment]
yes right

File: src/app/Analytics/SRE_Analytics/UsecaseSelection.tsx [hunk 1]
```diff
@@ -83,7 +83,10 @@ const UsecaseSelection = (props: { endTimeArray; setEndTimeArray; SREdata; setSR
       var namespace = data[0].kubernetes_objects[0].namespace;
       var name = data[0].kubernetes_objects[0].name;
       var type = data[0].kubernetes_objects[0].type;
-
+      var cluster_name = data[0].cluster_name;
+      var container_name = data[0].kubernetes_objects[0].containers[0].container_name;
+      // console.log(data[0].kubernetes_objects[0].containers[0].container_name);
+      // console.log(data);
```
Comment (dinogun) [inline src/app/Analytics/SRE_Analytics/UsecaseSelection.tsx:89]: Please remove commented out code

Comment (bhanvimenghani) [reply src/app/Analytics/SRE_Analytics/UsecaseSelection.tsx]: done!

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**